sale_report_hide_price/views/report_saleorder.xml

```diff
@@ -6,49 +6,38 @@
         <td t-if="display_discount" position="replace">
             <t t-if="doc.show_discount">
                 <td t-if="display_discount" class="text-right" groups="sale.group_discount_per_so_line">
-                    <span t-field="l.discount" />
+                    <span t-field="line.discount" />
                 </td>
             </t>
             <t t-if="not doc.show_discount">
                 <td class="text-right" groups="sale.group_discount_per_so_line">-N.A.-</td>
             </t>
         </td>
-        <xpath expr="//table[hasclass('table-condensed')]/tbody/t/t[2]/tr/td[3]" position="replace">
+        <xpath expr="//table[hasclass('table-sm')]/tbody/t[2]/tr/t/td[3]" position="replace">
             <t t-if="doc.show_price">
                 <td class="text-right">
-                    <span t-field="l.price_unit" />
+                    <span t-field="line.price_unit" />
                 </td>
             </t>
             <t t-if="not doc.show_price">
                 <td class="text-right">-N.A.-</td>
             </t>
         </xpath>
-        <xpath expr="//table[hasclass('table-condensed')]/tbody/t/t[2]/tr/td[last()-1]" position="replace">
+        <xpath expr="//table[hasclass('table-sm')]/tbody/t[2]/tr/t/td[last()]" position="replace">
             <t t-if="doc.show_price">
-                <td class="text-right" groups="sale.group_show_price_subtotal">
-                    <span t-field="l.price_subtotal"
-                          t-options='{"widget": "monetary", "display_currency": doc.pricelist_id.currency_id}' />
+                <td class="text-right" groups="account.group_show_line_subtotals_tax_excluded">
+                     <span t-field="line.price_subtotal" groups="account.group_show_line_subtotals_tax_excluded"/>
+                     <span t-field="line.price_total" groups="account.group_show_line_subtotals_tax_included"/>
```

Comment:
@JayVora-SerpentCS  in v12 Total Price(price_total) is included in same <td> where price_subtotal field is. Previously in v11  they were in different <td>.